doc/manual/calling-c-and-fortran-code.rst

```diff
@@ -561,6 +561,44 @@ Arrays of unknown size are not supported.
 
 In the future, some of these restrictions may be reduced or eliminated.
 
+SIMD (SSE, AVX)
+~~~~~~~~~~~~~~~
+
+On 64-bit x86, if a C/C++ routine has an argument or return value with an 
```

Comment:
Is it possible to organize this to void mentioning platform specific things in the general part? (e.g. `SIMD` instead of `SSE`/`AVX`, maybe only add a note that only x64 is supported since other archs haven't been implemented yet). Of course examples can still use whatever convenient to write and architecture specific limitation/features can still be added after the general part.